ketcher-autotests/tests/Indigo-Tools/Aromatize-Dearomatize/aromatize-dearomatize.spec.ts

```diff
@@ -378,4 +379,105 @@ test.describe('Aromatize/Dearomatize Tool', () => {
 
     expect(cmlFile).toEqual(cmlFileExpected);
   });
+
+  test('User can auromatize molecules with query parameters (not custom query, but only ordinary).', async ({
```

Comment:
I guess there should be 'Aromatize' instead of 'Auromatize'.

Reply:
Done.